vuepress/docs/next/tutorials/create/strapi-content.md

```diff
@@ -0,0 +1,83 @@
+# Strapi Content Management
+
+## Overview
+
+This tutorial demonstrates how to create and manage the content in your Strapi instance from the App Builder user interface.
+
+The Strapi Content Widget and Strapi Content List Widget enable the user to add or edit content and content lists, respectively, where it is mandatory to assign templates to selected content. The Strapi Content List Widget also allows content to be saved as a search. The following table compares widget capabilities.
```

Comment:
Not sure if it's the strange characteristics of these widgets but what is the relationship between adding/editing content & adding/editing content lists, to the requirement to assign templates to content. It is not clear if there is a reason why it  is mentioned in the same sentence.

maybe it's the 'where' that is vague, where is it mandatory?

Reply:
i have no clue why content and content lists had to be separated into two widgets, or why it's required to assign a template. it's like they're overengineering strapi, so you have to relearn how to use strapi in the context of entando. i know there must be (good) reasons but they were not communicated in notion or by NV. it is mandatory to add a template when you add or edit, on the page where the adding or editing is done. i can rephrase but not explain the construction/rules.